matrix-sdk/src/main/java/org/matrix/androidsdk/rest/model/URLPreview.java

```diff
@@ -55,35 +59,29 @@
 
     private boolean mIsDismissed;
 
-    /**
-     * Returns a dedicated parameter as a string
-     *
-     * @param paramName the parameter name
-     * @return the string value
-     */
-    private static String getAsString(Map<String, Object> map, String paramName) {
-        if (map.containsKey(paramName) && (map.get(paramName) instanceof String)) {
-            return (String) map.get(paramName);
-        }
-
-        return null;
-    }
-
     /**
      * Constructor
      *
      * @param map the constructor parameters
      */
-    public URLPreview(Map<String, Object> map) {
-        mDescription = getAsString(map, OG_DESCRIPTION);
-        mTitle = getAsString(map, OG_TITLE);
-        mType = getAsString(map, OG_TYPE);
+    public URLPreview(Map<String, Object> map, String url) {
```

Comment:
is it normal that verification tools do not complain about undocumented parameter? ;)

Reply:
It used to. I've checked in my configuration and it's enabled... Maybe an issue on the IDE.